src/plugins/data/public/search/search_interceptor.ts

```diff
@@ -52,6 +53,7 @@ export interface SearchInterceptorDeps {
   startServices: Promise<[CoreStart, any, unknown]>;
   toasts: ToastsSetup;
   usageCollector?: SearchUsageCollector;
+  uiActions: UiActionsStart;
```

Comment:
i would avoid here and add to the extended ppl and sql interceptors

Reply:
Can you clarify what this means?